src/api-calls.js

```diff
@@ -16,6 +16,7 @@ const fetchRooms = () => {
 }
 
 const postBooking = (selectedRoom, customer, date) => {
```

Comment:
I think this post looks good! If you wanted to keep your reference in showErrorMessage() in the domUpdates file since it is changing dom info, you could try setting up your api calls like this:

const postBooking = (selectedRoom, customer, date) => {
    return fetch ( // fetch call info
    )
    .then(response => handleError(response))

//function that is also in api calls:
const handleError = (response) => {
    if (!response.ok) {
        throw new Error ('<what ever you want your error message to be for a 422>');
    } else {
        return response.json();
    }
}

You would have just your fetch calls and this then statement after each one in your api file. Then in domUpdates or wherever you are invoking the fetch functions, you can add another then statement after the invocation where you do something with the fetch, and then add a catch where you show your error message.

//in domUpdates (probably inside another function)
postBooking()
    .then(response => //do something with response, probably a dom update function)
    .catch(error => domUpdates.displayErrorMessage(error))

Definitely not necessary to set it up like this, but this is how I went about keeping the domUpdate for the error inside my domUpdates file. I think it is just fine how you have it though (especially if it is all working fine), so just depends on what your preference is!